src/components/Pokedex2/PokemonPageContent.jsx

```diff
@@ -94,6 +135,19 @@ export const PokemonPageContent = ({ pokemon, pokemonNames }) => {
           <PokemonMovesetList moveset={pokemon.learnset.tutor} movesetPrefix="tutor" pokemonDexId={pokemon.id} />
         </PokemonAccordion>
       </Container>
+
+      <Container>
+        <Box style={{marginTop: '50px', marginBottom: '10px'}}>
+          <Typography variant="overline">
+            <b>Credits:</b>
```

Comment:
Add a new line here as it gets a little cramped with the title of the section and the content on the same line.